runtime/memprof.c

```diff
@@ -313,3 +342,103 @@ void caml_memprof_handle_postponed() {
   }
   caml_memprof_set_suspended(0);
 }
+
+static void set_trigger(value* trigger) {
+  caml_memprof_young_trigger = trigger;
+  caml_young_limit = caml_young_trigger < trigger ?
+    trigger : caml_young_trigger;
```

Comment:
Note: the largest value is picked because `caml_young_ptr` is decreasing.

Reply:
Yes. Do you really think this is worth a comment?

Reply:
As you feel. It's probably fine as it is, but making the assumption explicit in my comment helped me review the code (and maybe it could help others).

I'm not familiar with the minor GC so it wasn't immediate for me to check that assumption (the wording of the documentation comment in `runtime/minor_gc.c` suggests it, but does not clearly say it. The place where it is made clear is in `runtime/caml/memory.h`, which is not the first place I thought of looking.